android/MobileSdk/src/main/java/com/spruceid/mobile/sdk/CredentialsViewModel.kt

```diff
@@ -126,8 +126,12 @@ class CredentialsViewModel(application: Application) : AndroidViewModel(applicat
         }
     }
 
+
+    // We specify a default value for presentData, defaulting to QR presentation
+    // Having a default value for the presentData to this feels wrong, but is required
```

Comment:
"this feels wrong" is a bit too casual IMO

That being said, I think we're fine with such a small breaking change